src/lambda/graphql.ts

```diff
@@ -0,0 +1,24 @@
+import { ApolloServer } from 'apollo-server-lambda'
+import { schema } from '../api/schema'
+
+const server = new ApolloServer({
+  schema,
+  introspection: true,
+  context: ({ event, context }: any) => ({
+    headers: event.headers,
+    functionName: context.functionName,
+    aws: { event, context }
+  }),
+  // optional
+  engine: { apiKey: process.env.ENGINE_API_KEY }
```

Comment:
I had an issue when `apiKey` was not existing or invalid, there was a hanging connection and the API wouldn't shut down properly. Especially during integration tests. No need to fix I guess but I wanted to warn you :)

Reply:
Interesting! First I though of shutting down `engine` config entirely when the key was absent, but no issues problems arised. I'll probably do remove it, just in case.

Reply:
Fixed on 5a8fb4e51c3af360c4ba4bdcb725ced100024aad :wink: